packages/test_track_test_support/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   equatable: ^2.0.7
   meta: ^1.17.0
   random_string: ^2.3.1
-  test_track: ^0.2.0
+  test_track: any
```

Comment:
I don't love this, but it should make upgrading these packages easier. Before, we had to publish the client prior to the support package. With pub workspaces, I'm not even sure how this is _supposed_ to work, as even trying to bootstrap I get an error. But with `any` pub workspaces works as expected and we shouldn't need to do the "upgrade dance" described above.